cmake/shared/PACE_Version.cmake

```diff
@@ -27,8 +27,9 @@ formatted version string
 #]=======================================================================]
 
 set(${PROJECT_NAME}_FULL_VERSION "${PROJECT_VERSION}")
+string(TOUPPER "${${PROJECT_NAME}_RELEASE_TYPE}" ${PROJECT_NAME}_RELEASE_TYPE)
```

Comment:
why the change from lower case to caps?

Reply:
Just a usability thing, when you run CMake you can now do `cmake .. DHerbert_RELEASE_TYPE=RELEASE` or `cmake .. DHerbert_RELEASE_TYPE=release`

Reply:
aren't you defining an input variable in that conversion? Would it be cleaner to store the TOUPPERed variable in a "local" `_release_type` ?

Reply:
Aren't you redefining an input variable in that conversion? Would it be cleaner to store the TOUPPERed variable in a "local" scoped variable ?